index.yaml

```diff
@@ -5277,7 +6015,36 @@ entries:
       catalog.cattle.io/release-name: sriov
     apiVersion: v2
     appVersion: 1.0.0
-    created: "2021-10-18T13:41:50.77855-07:00"
+    created: "2021-11-19T11:49:26.033710053-08:00"
+    description: SR-IOV network operator configures and manages SR-IOV networks in
+      the kubernetes cluster
+    digest: 6d61497adfa17683860eaf7fcb0ac7a72862c5c818d9a8c364cb5dfe9ba3a788
+    home: https://github.com/k8snetworkplumbingwg/sriov-network-operator
+    icon: https://charts.rancher.io/assets/logos/sr-iov.svg
+    keywords:
+    - sriov
+    - Networking
+    kubeVersion: '>= 1.16.0-0'
+    maintainers:
+    - email: charts@rancher.com
+      name: Rancher Labs
+    name: sriov
+    sources:
+    - https://github.com/rancher/charts
+    type: application
+    urls:
+    - assets/rancher-sriov/sriov-100.0.1+up0.1.0.tgz
+    version: 100.0.1+up0.1.0
```

Comment:
the following three annotations are missing from this chart: 
```
catalog.cattle.io/kube-version
catalog.cattle.io/rancher-version
catalog.cattle.io/upstream-version
```

Reply:
This was no listed on new charts for 2.6.3.. it should definitely have these annotations as its user facing.

Reply:
Thank @jiaqiluo for updating that! These have now been added to the chart. Do we need QA to check it?

Reply:
There are two major changes to the chart:
- fix the helper function for default registry: https://github.com/rancher/charts/pull/1628
- add new annotations 

I think it is worthy to check if the app can be installed properly  in an air-gapped env